pkg/convert/jfr/parser.go

```diff
@@ -268,9 +267,13 @@ var generatedMethodAccessor = regexp.MustCompile("^(jdk/internal/reflect/Generat
 // org/example/rideshare/OrderService$$Lambda$669.0x0000000800fd7318.run
 var lambdaGeneratedEnclosingClass = regexp.MustCompile("^(.+\\$\\$Lambda\\$)\\d+[./](0x[\\da-f]+|\\d+)$")
 
+// libzstd-jni-1.5.1-16931311898282279136.so.Java_com_github_luben_zstd_ZstdInputStreamNoFinalizer_decompressStream
+var zstdJniSoLibName = regexp.MustCompile("^(libzstd-jni-\\d+\\.\\d+\\.\\d+-)(\\d+)(\\.so)$")
```

Comment:
we should definitely expand this to more jni libraries, not just libzstd

Reply:
./tmp/libamazonCorrettoCryptoProvider109b39cf33c563eb.so
./tmp/libamazonCorrettoCryptoProvider492aaa424fafd7e0.so

Reply:
I mean more like make it so that it would work with any arbitrary library

Reply:
I don't get it
This naming logic is somewhat app/lib specific.
For example pyroscope adds a sha1 to a library name, amazon 8byte hex, zstd some long integer.

Reply:
ah, I didn't realize that, I thought this was something standardized